sqla_yaml_fixtures/__init__.py

```diff
@@ -59,7 +59,10 @@ def _create_obj(ModelBase, session, store,
     :var values (dict): column:value
     '''
     # get reference to SqlAlchemy Mapper
-    model = ModelBase._decl_class_registry[model_name]
+    model = (
+        getattr(ModelBase, "_decl_class_registry")
+        or ModelBase.registry._class_registry
+    )[model_name]
```

Comment:
This doesn't actually seem to work, it will still fail. This does work:

```python
    # get reference to SqlAlchemy Mapper
    try:
        model = ModelBase._decl_class_registry[model_name]
    except AttributeError:
        model = ModelBase.registry._class_registry[model_name]
```